core/arch/arm/plat-imx/imx_pl310.c

```diff
@@ -51,10 +52,14 @@ void arm_cl2_config(vaddr_t pl310_base)
 
 	/* invalidate all cache ways */
 	arm_cl2_invbyway(pl310_base);
+#else
+	(void)pl310_base;
```

Comment:
Please use the `__maybe_unused` attribute for the parameter instead.

Reply:
Fix in next version